libraries/AP_GPS/AP_GPS.h

```diff
@@ -428,6 +428,9 @@ class AP_GPS
     static const char _initialisation_blob[];
     static const char _initialisation_raw_blob[];
 
+    static const uint64_t ms_per_week = 7000ULL*86400ULL;
+    static const uint64_t unix_offset = 17000ULL*86400ULL + 52*10*7000ULL*86400ULL - GPS_LEAPSECONDS_MILLIS;
```

Comment:
@WickedShell Comments Thank you.
1. The variable name "ms_per_week" used the original as it was.
2. The original description is "to be a const variable because # dififine in C ++ is different from C", it is in other manners.
Therefore,
I used it as it was.